display.js

```diff
@@ -49,21 +49,22 @@ function createExtracts(results) {
 function activatePage() {
     var spinner = createSpinner();
     RESULTSBOX.appendChild(spinner);
-    return getLocations().then(
+    getLocations().then(
         results => {
             spinner.style.display = "none";
             if (results.length != 0) {
                 RESULTSBOX.querySelector("h2").innerText =
                     "Here are the locations mentioned on the page.";
                 RESULTSBOX.appendChild(createExtracts(results));
-                results.forEach(x =>
-                    searchAndReplace(
-                        document.body,
-                        x.spot,
-                        x.lod.wikipedia,
-                        createExtract(x)
-                    )
-                );
+                results.forEach(x => {
+                    var dataDict = {
```

Comment:
might be better to not make variable name and just pass the dict itself?

Reply:
I have the dataDict there so we can pass only the stuff we need, not everything from x

Reply:
now that we changed the below, we can remove this variable?

Reply:
Yeah, go ahead